students/TinaB/lesson05/mailroom_part3.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python3
+"""
+$ chmod +x mailroom_part3.py
+"""
+import datetime
+import os
+
+DONORS_DICT = {"William Gates": [326892.24, 122, 22, 12],
+               "Mark Zuckerberg": [30, 60, 65982.55],
+               "Jeff Bezos": [52636.27],
+               "Paul Allen": [877.33, 22],
+               "Steven Hawking": [326892.24, 123, 123.33, 123, 123],
+               "Justin Timberlake": [999658.25, 1233, 123]}
```

Comment:
all caps are usually used for constants, since we are modifying donors dictionary it should be all lower case variable.